Source/WTF/wtf/Compiler.h

```diff
@@ -452,10 +452,19 @@
     __attribute__((__no_reorder__)) void name(void) { __asm__(""); }
 #endif
 
+#if !defined(NO_REORDER) && COMPILER(GCC)
+#define NO_REORDER \
+    __attribute__((__no_reorder__))
+#endif
+
```

Comment:
This is wrong place to put this.  This section is for DEBUGGER_ANNOTATION_MARKER.  See how REFERENCED_FROM_ASM does it.  Please fix before landing.